src/client/pages/registration/components/Atoms/RegistrationLink.jsx

```diff
@@ -0,0 +1,11 @@
+import css from './RegistrationLink.module.css';
+
+const RegistrationLink = () => {
+  return (
+    <a href="/teamproject-react-node-soyummy/signin" className={css.RegistrationLink}>
```

Comment:
I'm not sure if `href` is correct but keep it in mind while testing.
Rest ✅

Reply:
Did you check it? ❓

Reply:
I changed the href and I hope it will work 🤞